### ISSUE TITLE
missing globals.html with typedefs

### ISSUE THREAD

**Issue by cancerberoSgx** [steveush/foodoc#5]
Given the following file (test.js)

```
/**
 * Some class
 * @class
 * @global
 */
class A {
    /**
     * some method
     * @param {SomeType} p
     */
    method1(p){
    }
}
/**
 * @typedef {Object} SomeType
 * @property {String} someProp
 */
```

if you generate documentation with the command `jsdoc test.js` (no foodoc) everything is OK, in A.html, you have a link to SomeType that goes to global.html#SomeType and it works

if you generate jsdocs with foodoc (command `jsdoc test.js -t node_modules/foodoc/template/`  then in A.html there is the same link that goes to the same location (global.html#SomeType) but the file globals.html is missing. 

The workaround I'm currently doing is generating both documentations so at least I have the globals.html file from the original template, i.e: 
`jsdoc test.js && jsdoc test.js -t node_modules/foodoc/template/`

Sorry I don't have the time to investigate this issue further... Thanks

**Comment by joshua-golub** [comment]
@cancerberoSgx, did you ever find a solution to this? I am having the same problem -- `foodoc` isn't generating the documentation for my global functions. My previous template (`ink-docstrap`) _did_.

**Comment by cancerberoSgx** [comment]
the workaround i found is compiling twice, one using foodoc and the other without it - so the global page is generated but with the old original template, something like this: 

```sh
    node node_modules/jsdoc/jsdoc.js -c jsdoc-extlayer-frontend-config.json
    node node_modules/jsdoc/jsdoc.js -c jsdoc-extlayer-frontend-config.json -t node_modules/foodoc/template
```

**Comment by Dhyey** [comment]
Any updates on this ? I too can't find page that documents all my typedefs.

**Comment by cancerberoSgx** [comment]
I will try to make a PR for this since this template is very cool and this is a serious issue. will keep you posted

**Comment by cancerberoSgx** [comment]
So there's my PR - is ugly but better than nothing.I guess you can work with my clone until author fix it or merge it.: https://github.com/cancerberoSgx/foodoc

**Comment by steveush** [comment]
Hi Guys,

Apologies this took so long to get to, the PR notification from @cancerberoSgx finally got my arse in gear and I think I have resolved this issue in the latest version 0.0.9. Please try it out and let me know.

Thanks

**Comment by cancerberoSgx** [comment]
just tried it in my big reference API full of global datatypes . Good job! keep it up!
